server/src/test/java/org/apache/uniffle/server/buffer/ShuffleBufferManagerTest.java

```diff
@@ -567,7 +568,25 @@ public void shuffleFlushThreshold() throws Exception {
 
     StorageManager storageManager = StorageManagerFactory.getInstance().createStorageManager(conf);
     ShuffleFlushManager shuffleFlushManager = new ShuffleFlushManager(conf,
-        "serverId", mockShuffleServer, storageManager);
+        "serverId", mockShuffleServer, storageManager) {
+
+      @Override
+      protected void startEventProcesser() {
+        // do nothing
+      }
+
+      @Override
+      public void processEvents() {
```

Comment:
I think we shouldn't override `processEvents()`, because if `ShuffleFlushManager#processEvents()` is changed, the test will pass no matter if the change is correct.